replay-events/.gitignore

```diff
@@ -0,0 +1,5 @@
+*
+!.gitignore
+!docker-entrypoint.sh
+!Dockerfile
+!replay.sh
```

Comment:
This feels a little backwards to me in that it will need to be updated every time a file needs to be added. Why not just ignore the files that actually need to be ignored (e.g. *.log)? It doesn't appear that the directory is going to have an arbitrary number of files with unpredictable names/extensions. There might be a good reason for this, so feel free to correct me if I'm missing the point.

This applies to other gitignore files in the PR as well.

Reply:
I do this precisely because I didn't want to make it possible to accidentally include log files with personal information in a git commit. (Which seems pertinent given what happened recently...) Since we don't actually want to track many files anyway, explicitly adding the few we do want to track here serves as a measure of protection, especially given the nature of the log files being generated.

If this repo were to start getting used more widely, or we were planning to expand it significantly, I'd agree it may not be helpful. But maybe we reevaluate at that point, rather than now?

Reply:
👍